### ISSUE TITLE
Personal Info Page - Add Full Form and Text

### ISSUE THREAD

**Issue by P0NDER0SA** [MTS-STM/DTS-Scheduler#73]
This includes adding all fields - Application Number, Email and Email Confirm.  also has a checkbox for accessibility requirement.  Validation should be done.  Text should be added. Nav should be done.  Session Management should be done.

- [x] Unit Tests
- [x] Accessibility(glenn)
- [x] Government Web Usability (Common look and Feel)(Marco)
- [x] Mobile friendly(Marco)
- [x] clean up locals(glenn)
- [x] french text(glenn)
- [x] validation(glenn)
- [ ] Task 3

**Comment by newcitywave** [comment]
some of the french text may need to be reviewed, what i added is from google translate.
